SRT: Updating libsrt to version 1.5.4

artem-smorodin-dacast opened the pull request:
This is an attempt to fix #4587

winlinvip commented:
Thank you for your PR, but 1.5.5 is RC, which is not stable version, please upgrade to latest stable version.

artem-smorodin-dacast commented:
@winlinvip Can we go ahead and use RC since it includes [improved blocking](https://github.com/Haivision/srt/pull/3189)? As well as many other useful things.

If not, I will change the version to 1.5.4 and update this PR.

I look forward to hearing your opinion.

artem-smorodin-dacast committed:
updating libsrt version to v1.5.4

artem-smorodin-dacast commented:
@winlinvip Since this does not fix #4587, it seems pointless to use RC, so I updated this PR and changed libsrt version to 1.5.4 (latest stable release).

winlinvip commented:
If it doesn't fix #4587, why should upgrade to libsrt 1.5.4?

winlinvip commented:
Thank you, but no need to upgrade to 1.5.4, because it doesn't fix #4587
